ui/app/routes/vault/cluster/dashboard.js

```diff
@@ -16,6 +16,8 @@ export default class VaultClusterDashboardRoute extends Route.extend(ClusterRout
   @service version;
 
   async getVaultConfiguration() {
+    if (!this.namespace.inRootNamespace) return null;
```

Comment:
Yes! Since we're checking to see if the user is not in the root namespace before doing our queries it doesn't make an api call. I think I checked the token use case, but if you can help double-check for me that would amazing!! @Monkeychip